pkg/redirectpolicy/manager.go

```diff
@@ -372,7 +372,11 @@ func (rpm *Manager) getAndUpsertPolicySvcConfig(config *LRPConfig) {
 
 	case svcFrontendSinglePort:
 		// Get service frontend with the clusterIP and the policy config (unnamed) port.
-		ip := rpm.svcCache.GetServiceFrontendIP(*config.serviceID, lb.SVCTypeClusterIP)
+		ip, err := rpm.svcCache.GetServiceFrontendIP(*config.serviceID, lb.SVCTypeClusterIP)
+		if err != nil {
+			log.Debugf("service information not available yet for policy %v", *config)
```

Comment:
Structured logging would be preferred over dynamic formatting
```suggestion
			log.WithField("...", config).Debug("service information not available yet for policy")
```

Reply:
This log message is redundant so I've removed it.